tests/foreman/cli/test_contentview.py

```diff
@@ -3892,50 +3455,26 @@ def test_positive_remove_cv_version_from_multi_env_capsule_scenario(
             }
         )
         Repository.synchronize({'id': custom_yum_repo['id']})
-        # Setup a puppet repo
-        puppet_product = cli_factory.make_product({'organization-id': module_org.id})
-        puppet_repository = cli_factory.make_repository(
-            {
-                'content-type': 'puppet',
-                'product-id': puppet_product['id'],
-                'url': settings.repos.puppet_0.url,
-            }
-        )
-        Repository.synchronize({'id': puppet_repository['id']})
-        puppet_modules = PuppetModule.list(
-            {'repository-id': puppet_repository['id'], 'per-page': False}
-        )
-        assert len(puppet_modules) > 0
-        puppet_module = puppet_modules[0]
-        # Setup a docker repo
         docker_product = cli_factory.make_product({'organization-id': module_org.id})
         docker_repository = cli_factory.make_repository(
             {
                 'content-type': 'docker',
-                'docker-upstream-name': 'busybox',
+                'docker-upstream-name': constants.CONTAINER_UPSTREAM_NAME,
```

Comment:
please provide information, why docker upstream name is changing here, respectively why `busybox` was used before

Reply:
I'm assuming `busybox` was used before because it returns content after syncing.  As I was testing this, I found that that's no longer the case and using `library/busybox` does return content after syncing and switched over to using the new name change to better reflect the update.